### DISCUSSION TITLE
Biding in other Querys

### DISCUSSION THREAD

**Discussion by Kakovisk** [Budibase/budibase#7611] [Help]
In a form I'm editing I want the first field to be filled with data from a sql query.

I added the Data Provider Component But unfortunately it does not appear in the options of the BINDABLE VALUES of Default Value of Text Field or Number Field.

What Happen?

In this situation I will need to search in 3 querys diferents with rules especifics.

**Comment by melohagan** [comment]
Hi @Kakovisk 

Would you mind sharing some screenshots please? I'm not sure what the issue is.
